tests/storage/cephfs/conftest.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+@pytest.fixture(scope='session')
+def host_with_ceph(host):
+    assert not host.file_exists('/usr/sbin/mount.ceph'), \
+        "mount.ceph must not be installed on the host at the beginning of the tests"
+    host.yum_install(['centos-release-ceph-jewel'], enablerepo="base,extras", save_state=True)
+    host.yum_install(['ceph-common'], enablerepo="base,extras")
+    yield host
+    # teardown
+    host.yum_restore_saved_state()
+
+@pytest.fixture(scope='session')
+def cephfs_device_config(sr_device_config):
+    if sr_device_config is not None:
+        # SR device config from CLI param
+        config = sr_device_config
```

Comment:
Are we sure `sr_device_config` has been properly parsed here?

Reply:
Yes, see the definition in the root conftest.py